Sources/Yams/Constructor.swift

```diff
@@ -185,50 +187,94 @@ extension Double: ScalarConstructible {
     }
 }
 
+extension Float: ScalarConstructible {
```

Comment:
Do you think it'd be possible to generalize this to the `FloatingPoint` protocol? I ran into several issues while trying to do it, but if you can find a way, seems like that'd be pretty powerful.

Reply:
I'll try.